server/start-dev.js

```diff
@@ -2,7 +2,7 @@
 
 console.log('start-dev process.env.NODE_ENV', process.env.NODE_ENV);
 
-const path = require('path');
+const path = require('path').posix;
```

Comment:
Using the `posix` version to get the glob patterns to work. Output with the `verbose` option with `nodemon`:

Before:

```
Nodemon logs: detail: ignoring: .\.git/**/* .\dist\**\!(manifest.json)
```

After:

```
Nodemon logs: detail: ignoring: .\.git/**/* dist/**/!(manifest.json)
```

I'm curious if the `.git` pattern works here since it has the same backslash problem but 🤷, it just means extra retarts and I haven't noticed anything. How much git operations are you ever doing for this to matter anyway.